package.json

```diff
@@ -27,8 +27,8 @@
     "install:web": "yarn install:example && cd website && yarn --ignore-engines",
     "install-and-start": "node ./scripts/install-and-start",
     "fast-test": "npm run test-node && npm run test-browser",
-    "test-browser": "NODE_ENV=test babel-tape-runner -r ./test/setup-browser-env.js ./test/browser.js | tap-spec",
-    "test-node": "NODE_ENV=test babel-tape-runner ./test/node.js | tap-spec",
```

Comment:
did you remove this on purpose or was just leftover from troubleshooting some tests?

Reply:
added back